### DISCUSSION TITLE
UnoCSS support

### DISCUSSION THREAD

**Discussion by oott123** [themeselection/flyonui#4] [Ideas / Request new feature]
FlyonUI is amazing! If we can support [UnoCSS](https://unocss.dev), it will be more amazing!

**Comment by PruthviPraj00** [comment]
Hello @oott123,

Thank you for your feedback! We're excited to hear you love FlyonUI. Supporting UnoCSS is a fantastic suggestion, and we're actively considering it for a future update!
